tests/unit/test-openai-reasoning-payload.py

```
@@ -0,0 +1,20 @@
+import importlib.util, json
+from unittest.mock import patch
+from pathlib import Path
+path=Path(__file__).resolve().parents[2]/"scripts/helpers/openai-compatible-agent.py"
+spec=importlib.util.spec_from_file_location("agent",path)
+mod=importlib.util.module_from_spec(spec); spec.loader.exec_module(mod)
```

Ruff (0.15.21)

[error] Multiple statements on one line (semicolon)

(E702)